Tools/TestWebKitAPI/Tests/WebKit/AdvancedPrivacyProtections.mm

```diff
@@ -39,9 +39,11 @@
 #import "UserInterfaceSwizzler.h"
 #import <WebCore/LinkDecorationFilteringData.h>
 #import <WebKit/WKErrorPrivate.h>
+#import <WebKit/WKWebView.h>
```

Comment:
Nit - I don't think you need this import, since we already import the WebKit umbrella header transitively.